framework/src/modules/governance/stores/index.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { BaseStore } from '../../base_store';
+
+export interface IndexStoreData {
+	nextIndex: number;
+	nextOutcomeCheckIndex: number;
+	nextQuorumCheckIndex: number;
+}
+
+export const indexSchema = {
+	$id: '/governance/store/index',
+	type: 'object',
+	required: ['nextIndex', 'nextOutcomeCheckIndex', 'nextQuorumCheckIndex'],
+	properties: {
+		nextIndex: {
+			dataType: 'uint32',
+			fieldNumber: 1,
+		},
+		nextOutcomeCheckIndex: {
+			dataType: 'uint32',
+			fieldNumber: 2,
+		},
+		nextQuorumCheckIndex: {
+			dataType: 'uint32',
+			fieldNumber: 3,
+		},
+	},
+};
+
+export class IndexStore extends BaseStore<IndexStoreData> {
+	public schema = indexSchema;
+}
```

Comment:
Following @shuse2 [suggestion](https://github.com/LiskHQ/lisk-sdk/pull/8888#discussion_r1302576852), we have renamed `IndexStore` into `ProposalIndexStore` and you can find the foundation in `proposal_index.ts` file.